fe/fe-core/src/main/java/com/starrocks/leader/SpreadRunningTaskRunProgress.java

```diff
@@ -0,0 +1,67 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.leader;
+
+import com.starrocks.common.FeConstants;
+import com.starrocks.common.util.LeaderDaemon;
+import com.starrocks.scheduler.TaskRun;
+import com.starrocks.scheduler.TaskRunManager;
+import com.starrocks.scheduler.persist.RunningTaskRunProgressInfo;
+import com.starrocks.server.GlobalStateMgr;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SpreadRunningTaskRunProgress extends LeaderDaemon {
```

Comment:
TaskRunStateSynchronizer may be better name?

Reply:
ok